### PR TITLE
Ports Medibot Tipping

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Ports:

- yogstation13/Yogstation#8917
- tgstation/tgstation#51409

A while ago I read a book about people and robots and emotions, Alone Together by Sherry Turkle, and there's one passage that talks about people's reactions to robots simulating distress, specifically with how Furbies cry when you hold them upside down.

A person is asked to invert three creatures: a Barbie doll, a Furby, and a biological gerbil. [Freedom] Baird's question is simple: "How long can you hold the object upside down before your emotions make you turn it back?"

People are surprised by how upset they get in this theater of distress. And then they get upset that they are upset. They often try to reassure themselves, saying things like, “Chill, chill, it’s only a toy!” They are experiencing something new: you can feel bad about yourself for how you behave with a computer program. Adults come to the upside-down test knowing two things: the Furby is a machine and they are not torturers. By the end, with a whimpering Furby in tow, they are on new ethical terrain.

Ethics memes aside, the idea off people being caught off guard at their guilt for causing harm to inanimate objects is really funny to me, and feels extra relevant to SS13. Considering people already regularly get in fights and maim each other over harming station pets, I picture this being a great source of arguments and fights ICly over what torturing a cute lil' medbot says about you.

tl;dr this PR lets you tip over medbots which prevents them from moving or healing, and they'll slowly get more and more distressed and desperate asking for help from people around them. After a few minutes, if no one helps right them, they'll manage to right themselves up and snitch on whoever tipped them over to medical so the psychologist can keep an eye on them.

## Why It's Good For The Game
Gives the psychologist more ammo to call people crazy, gives bored assistants another way to seek attention, provides source of minor IC conflict

## Testing Photographs and Procedure

<details>

<summary>Screenshots&Videos</summary>

image

Put screenshots and Videos documenting testing and execution of intended behaviors here

</details>

## Changelog
:cl: Ivniinvi, swissloaf, Ryll-Ryll
add: You can now tip over medibots, and they will become increasingly more distressed.
/:cl:

### CODE REVIEW

**Comment by Archanial** [comment]
Epic embed fail

File: code/modules/mob/living/simple_animal/bot/medbot.dm [hunk 1]
```diff
@@ -230,10 +244,95 @@ GLOBAL_VAR(medibot_unique_id_gen)
...
+			speak("PSYCH ALERT: Crewmember [tipper_name] recorded displaying antisocial tendencies by torturing bots in [get_area(src)]. Please schedule psych evaluation.", radio_channel)
+			set_right() // strong independent medbot
+
+	if(prob(tipped_status))
+		do_jitter_animation(tipped_status * 0.1)
+
+	if(messagevoice)
+		var/message = pick(messagevoice)
+		speak(message)
+		playsound(src, messagevoice[message], 70)
```
Comment (Archanial) [inline code/modules/mob/living/simple_animal/bot/medbot.dm:306]: How often does it happen? It looks like every tick

Comment (Ivniinvi) [reply code/modules/mob/living/simple_animal/bot/medbot.dm:306]: It only fires when it handles automated action

**Review by Archanial — APPROVED** [review]

**Comment by Sweettoothart** [comment]
How will this affect pais inside of the medibots? Can pai medibots be tipped as well?

**Pull request merged**